app/src/main/java/com/kickstarter/viewmodels/DashboardRewardStatsRowHolderViewModel.java

```diff
@@ -0,0 +1,112 @@
+package com.kickstarter.viewmodels;
+
+import android.support.annotation.NonNull;
+import android.util.Pair;
+
+import com.kickstarter.libs.ActivityViewModel;
+import com.kickstarter.libs.Environment;
+import com.kickstarter.libs.utils.NumberUtils;
+import com.kickstarter.libs.utils.PairUtils;
+import com.kickstarter.libs.utils.StringUtils;
+import com.kickstarter.models.Project;
+import com.kickstarter.services.apiresponses.ProjectStatsEnvelope;
+import com.kickstarter.ui.viewholders.CreatorDashboardRewardStatsRowViewHolder;
+
+import rx.Observable;
+import rx.subjects.PublishSubject;
+
+import static com.kickstarter.libs.rx.transformers.Transformers.combineLatestPair;
+
+public interface DashboardRewardStatsRowHolderViewModel {
+
+  interface Inputs {
+    void projectAndRewardStats(Pair<Project, ProjectStatsEnvelope.RewardStats> projectAndRewardStats);
+  }
+
+  interface Outputs {
+    /* current project being viewed */
+    Observable<Project> currentProject();
+
+    /* project and the amount pledged for this reward */
+    Observable<Pair<Project, Float>> projectAndPledgedForReward();
+
+    /* string number of backers */
+    Observable<String> rewardBackerCount();
+
+    /* minimum for reward */
+    Observable<String> rewardMinimum();
+
+    /* percent of the total that came from this reward */
+    Observable<String> percentageOfTotalPledged();
+
+  }
+
+  final class ViewModel extends ActivityViewModel<CreatorDashboardRewardStatsRowViewHolder> implements
+    Inputs, Outputs {
+    public ViewModel(final @NonNull Environment environment) {
+      super(environment);
+
+      this.currentProject = projectAndRewardStats
+        .map(PairUtils::first);
+
+      this.rewardStats = projectAndRewardStats
+        .map(PairUtils::second);
+
+      this.floatAmountPledgedForReward = rewardStats
```

Comment:
is `floatAmountPledgedForReward` being used by the view? if not, we can just make this an intermediate observable within the view model, i.e.

```java
final Observable<Float> floatAmountPledgedForReward = ...
```